slices/grep.yaml

```diff
@@ -9,10 +9,11 @@ slices:
       /bin/grep:
 
   deprecated:
-    # These are shell scripts requiring a symlink from /usr/bin/bash to /bin/sh
+    # These are shell scripts requiring a symlink from /usr/bin/dash to /bin/sh
     # See: https://manpages.ubuntu.com/manpages/jammy/en/man1/grep.1.html
     essential:
-      - bash_bins
+      - dash_bins
+      - grep_bins
```

Comment:
out of curiosity, Why do you need grep_bins? aren't these purely relying on `dash` and `dash` only?

Reply:
They just shell out to `grep` so they'll fail if `grep_bins` isn't installed.

Reply:
This is what `egrep` is on my jammy instance:

```
#!/bin/sh
exec grep -E "$@"
```

Reply:
Noted!